scipy/signal/bsplines.py

```diff
@@ -17,5 +17,5 @@ def __dir__():
 
 def __getattr__(name):
     return _sub_module_deprecation(sub_package="signal", module="bsplines",
-                                   private_modules=["_bsplines"], all=__all__,
+                                   private_modules=["_spline_filters"], all=__all__,
```

Comment:
same (should move to https://github.com/scipy/scipy/commit/48dfec69357e1f0983426ddd76e295e077cfa452)

Reply:
done